ncs_con/obj_msg[] objs
